tests/pytests/data/lsb_0296962438_0x53c_eng/lsb_0296962438_0x53c_eng-spiced_2.xsp

```diff
@@ -0,0 +1,278 @@
+'-9177E2D69AFC6^0'
+'B2497F51237998^-3'
+'-29AD86E21EAE2A^-6'
+'-109044C967CB1A^9'
+'1100A0A9164CA3^6'
+'94210295FA8E5^3'
+'-72FDDAB32180F^0'
+'-4BCC77DC75FC5^-3'
+'12F0EB92CA7FE^-5'
+'-67EEEDB012BCBC^8'
+'-1CEAB1C413152D^6'
+'3A17A06D6A8238^3'
+'7F300DF2565778^-1'
+'-4DE156A15BA13C^-3'
+'5C6D19D3DFDB38^-6'
+'1CF5CB4^8'
+'2A3^6'
+'14^2'
+'1^1'
+END_ARRAY 2 24
+BEGIN_ARRAY 3 39
+'DE-0433LE-0433                          '
+'1D1533070890FD^8'
+'1D1533115934D5^8'
+'A'
+'0'
+'1'
+'2'
+39
+'1D156F4^8'
+'A8C^5'
+'7C2C8974688124^5'
+'BB81A2A56C5F8^3'
+'-167B59F54C4D11^2'
+'2BBB0C5C7B37F8^0'
+'ECDB7C7D083708^-2'
+'-41C9354D8B189C^-3'
+'-10873873EFBDC2^-3'
+'-25EFEE3FDA8F1E^-4'
+'-3F8F007ED5F8C^-5'
+'-47F00363930588^-6'
+'-1A416F739567E5^-7'
+'65E859CEC8BB28^4'
+'1AD4088C476137^4'
+'5019236C1FEAC8^1'
+'-16EF8F7A335A2^0'
+'17806612058433^-1'
+'1221AA3416A7DD^-2'
+'12089CF44D8054^-3'
+'A8ED8145966928^-5'
+'-FA4E301B157098^-6'
+'-49E23D1327444C^-6'
+'-9B6E7DE7350998^-7'
+'-345FB9B8E7B2BC^4'
+'B4B26FE94E58E8^3'
+'2AF420352A2E2E^1'
+'-C85724F3D31858^-1'
+'A3BECACF30B2F^-2'
+'9AE109119AAEC^-3'
+'B6DC845D407FE8^-4'
+'998144C3ADF52^-5'
+'-1C5387BD1CEB58^-6'
+'-2003910C77E128^-6'
+'-505FD93E3132A4^-7'
+'1D0AE34^8'
+'1518^6'
+'23^2'
+'1^1'
+END_ARRAY 3 39
+BEGIN_ARRAY 4 73
+'pvshort.nio                             '
+'1D1533070890FD^8'
+'1D1533115934D5^8'
+'-62'
+'9'
+'1'
+'1'
+73
+'1D154C57AAC4C6^8'
+'1ACDD15CC0BFA^4'
+'29ADA3A49BEA1^4'
+'2ED4D193719EF^4'
+'309DD59BA1E8F8^4'
+'313C29A51B5BDC^4'
+'3173038A4983B^4'
+'B335FE49CF468^4'
+'22F408C1CA9648^5'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'-52D4F4AC701F94^5'
+'28DFF8991CF332^0'
+'1C20D9AF96F2EF^7'
+'-D55DD9E7B56BD^1'
+'7575DC198DF794^6'
+'-37BD608B3A878E^1'
+'-22F38DF5B2525^-8'
+'14BF9E218FF442^-A'
+'-11BF66B15D59F6^-E'
+'10B151C56E328^-10'
+'3FA^-12'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'50C63313AF7D1C^-8'
+'-33CEBDBA113CEE^-A'
+'-282B126369271^-D'
+'-5FE2540D2A62CC^-F'
+'-B359C^-11'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'179F35152728F4^-8'
+'-17F1EA8796870A^-A'
+'-8BC2A1D9C78238^-E'
+'-18FD3257D23542^-F'
+'-2FB1C^-11'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'0^0'
+'6^1'
+'4^1'
+'5^1'
+'4^1'
+'1D154C57AAC4C6^8'
+'1^1'
+END_ARRAY 4 73
+TOTAL_ARRAYS 4
+ ~NAIF/SPC BEGIN COMMENTS~
+; /Users/astamile/testData/ale/leisa/lsb_0296962438_0x53c_eng/lsb_0296962438_0x53c_eng-spiced_2.bsp LOG FILE
+
+; Created 2022-06-10/07:27:59.00.
+;
+; BEGIN SPKMERGE COMMANDS
+
+LEAPSECONDS_KERNEL = /Volumes/pkgs/isis3/isis_data/base/kernels/lsk/naif0012.tls
+
+SPK_KERNEL = /Users/astamile/testData/ale/leisa/lsb_0296962438_0x53c_eng/lsb_0296962438_0x53c_eng-spiced_2.bsp
+SOURCE_SPK_KERNEL = /Volumes/pkgs/isis3/isis_data/newhorizons/kernels/spk/nh_recon_pluto_od122_v01.bsp
+    INCLUDE_COMMENTS = NO
+    BODIES           = 999, 9, 10, -98
+    BEGIN_TIME       = 2015 JUN 18 19:41:55.849
+    END_TIME         = 2015 JUN 18 19:42:06.164
```

Comment:
Removed light time correction and upped padding to 35 to get this in the range for `2015 JUN 18 19:41:26.054` and still came up with the same error:

`Insufficient ephemeris data has been loaded to compute the state of -98 (NH_SPACECRAFT) relative to 0 (SOLAR SYSTEM BARYCENTER) at the ephemeris epoch 2015 JUN 18 19:41:26.054.`